api/src/app.py

```diff
@@ -5,49 +5,41 @@
 import traceback
 from pathlib import Path
 from zipfile import ZipFile
-
+import uvicorn
 import click
 import emoji
 from dmss_api.exceptions import ApiException
-from flask import Flask
 
 from repository.repository_exceptions import ImportReferenceNotFoundException
-from services.job_service import JobService
 
+from fastapi import APIRouter, FastAPI
 from config import config
-from controllers import blueprints, entity, system, jobs
+
 from services.dmss import dmss_api
 from use_case.import_package import import_package_tree, package_tree_from_zip
 from utils.create_application_utils import zip_all
 from utils.logging import logger
+from store_headers_middleware import StoreHeadersMiddleware
 
+prefix = "/api"
```

Comment:
```suggestion
prefix = "/v1/api"
```
Perhaps we should use version prefixing here as well? (as we already do in DMSS)